corehq/apps/userreports/ui/forms.py

```diff
@@ -90,6 +90,12 @@ def clean_table_id(self):
             raise ValidationError(
                 _('Table id is too long. Your table id and domain name must add up to fewer than 40 characters')
             )
+        for src in self.instance.by_domain(self.domain):
+            if src.table_id == table_id and src.get_id != self.instance.get_id:
```

Comment:
I was thinking we usually use `_id`, but then PEP8 was telling me not to access private instance variables, so I caved :)